storybook/stories/playground/zoom-out/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { registerCoreBlocks } from '@wordpress/block-library';
+import { useDispatch } from '@wordpress/data';
+import {
+	BlockEditorProvider,
+	BlockCanvas,
+	store as blockEditorStore,
+	BlockList,
+} from '@wordpress/block-editor';
+import { __dangerousOptInToUnstableAPIsOnlyForCoreModules } from '@wordpress/private-apis';
+import { parse } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import editorStyles from '../editor-styles';
+// eslint-disable-next-line @wordpress/dependency-group
+import contentCss from '!!raw-loader!../../../../packages/block-editor/build-style/content.css';
+import { pattern } from './pattern';
+
+// Temporary hack to access private APIs before stabilizing zoom level.
+const { unlock } = __dangerousOptInToUnstableAPIsOnlyForCoreModules(
+	'I acknowledge private features are not for use in themes or plugins and doing so will break in the next version of WordPress.',
+	'@wordpress/edit-site'
+);
```

Comment:
Ideally setZoomLevel shouldn't be a private API, I think a block-editor setting makes sense for the zoom level setting but I'm keeping that separate. 

Also, it seems the value of zoom-level doesn't have any impact, it should probably correspond to the applied scale.